lentille-gitlab/src/Lentille/GitLab/MergeRequests.hs

```diff
@@ -260,20 +260,21 @@ transformResponse host result =
           where
             toNotesNodes (ProjectMergeRequestsNodesNotesNoteConnection nodes) = cleanMaybeMNodes nodes
             toMRComment :: ProjectMergeRequestsNodesNotesNodesNote -> MRComment
-            toMRComment (ProjectMergeRequestsNodesNotesNodesNote nId author' commentedAt ntype) =
+            toMRComment (ProjectMergeRequestsNodesNotesNodesNote nId author' commentedAt ntypeM) =
               let ProjectMergeRequestsNodesNotesNodesAuthorUserCore author'' = author'
-                  commentType = getCommentType ntype
+                  commentType = getCommentType ntypeM
                   NoteID noteIDT = nId
                in MRComment
                     { coId = sanitizeID noteIDT,
                       coAuthor = toIdent host author'',
                       coAuthoredAt = commentedAt,
                       coType = commentType
                     }
-            getCommentType ntypeM = case ntypeM of
+            getCommentType ntypeM' = case ntypeM' of
```

Comment:
nit: using `LambdaCase` we can avoid those un-necessary binding:

```suggestion
            getCommentType = \case
```

Reply:
oh great let's use that then !